src/ring_theory/free_comm_ring.lean

```diff
@@ -1,10 +1,11 @@
 /-
 Copyright (c) 2019 Kenny Lau. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Kenny Lau
+Authors: Kenny Lau, Johan Commelin
 -/
 
 import group_theory.free_abelian_group data.equiv.algebra data.equiv.functor data.polynomial
+import data.real.irrational
```

Comment:
For what do we need this file? It seems like a surprising import to me.

Reply:
It's part of a clever proof of `is_supported_of {p} {s : set α} : is_supported (of p) s ↔ p ∈ s`